specification/ai/Azure.AI.Projects/evaluation-results/models.tsp

```diff
@@ -0,0 +1,74 @@
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "../common/models.tsp";
+import "../main.tsp";
+
+using TypeSpec.OpenAPI;
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Azure.AI.Projects;
+
+@doc("Evaluation Result resource Definition")
+@discriminator("ResultType")
+@Rest.resource("evaluationresult")
```

Comment:
would 
```suggestion
@Rest.resource("evaluationResults")
```

be better? To align with other resource names being plural? Also camel casing it means autogenerated clients can use something like:

```python
client.evaluation_results(...)
```

Instead of 

```python
client.evaluationresults(...)
```